pnpm-lock.yaml

```diff
@@ -4575,6 +4575,7 @@ packages:
       { integrity: sha512-+dQSyqPh4x1hlO1swXBiNb2HzTDN1I2IGLQx1GrBuiqFJfoMrnZWwVmatvSiO+Iz8fBUnf+lekwNo4c2LlXItg== }
     engines: { node: ">=14" }
     hasBin: true
+    dev: false
```

Comment:
what is it?

Reply:
I wasn't able to commit my changes due to some restrictions with prettier, so I had to install and check prettier cases 
And that made this change

Reply:
like? what did you typed?

Reply:
i wrote 
pnpm install --save-dev prettier
pnpm prettier --write
pnpm prettier --check

these were the sol i got when i researched about the error.


Errors:
![image](https://github.com/priyankarpal/ProjectsHut/assets/100708686/a354fba4-90d9-45fa-97e6-5b93d642a9d7)
![image](https://github.com/priyankarpal/ProjectsHut/assets/100708686/2da18bf3-8537-446f-84ed-e1675b23e49d)

Reply:
no no, don't do these mistakes in future. Please read contributing guidelines we already mentioned about this problem. if you face this type of warn related problem you need to format files using `pnpm format` after that stage all changes using `git add --all` then commit & push

Reply:
Ok I will redo